client/src/Annotator/Paper/Utils/pathUtils.tsx

```diff
@@ -0,0 +1,95 @@
+/**
+ * Utils methods to modify exisitng paper.shapes
+ */
+import paper from 'paper';
+
+import { Annotation } from '../../../common/types';
+
+export const create = (a: Annotation, color?: string) => {
+    const item = new paper.CompoundPath({});
+    item.name = 'Annotation_' + a.id;
+    item.remove(); // newly created item is removed from project
+    item.visible = false; // and not visible
+
+    let center = new paper.Point(a.width / 2, a.height / 2);
+    let children: Array<paper.Path> = [];
+    a.segmentation.forEach(polygon => {
```

Comment:
one letter variable name aren't good. I assume that your IDE has autocompletion so a longer name not gonna hurt.